src/xtopcom/xbasic/src/xbasic_size.cpp

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2017-2021 Telos Foundation & contributors
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "xbasic/xbasic_size.hpp"
+
+NS_BEG1(top)
+
+size_t get_size(const std::string & str) {
```

Comment:
need add support for GCC with version higher than 4.8.5

Reply:
don't know the exact gcc version boundary of different string realization. will add support after we use higher gcc version.